ionoscloud/resource_k8s_node_pool.go

```diff
@@ -269,10 +270,10 @@ func resourcek8sNodePoolCreate(ctx context.Context, d *schema.ResourceData, meta
 					lan.Id = &lanID
 					addLan = true
 				}
-				if lanDhcp, lanDhcpOk := d.GetOk(fmt.Sprintf("lans.%d.dhcp", lanIndex)); lanDhcpOk {
-					lanDhcp := lanDhcp.(bool)
-					lan.Dhcp = &lanDhcp
-				}
+
```

Comment:
Please also update the check in the `Update` function below, where there's still 
```
if lanDhcp, lanDhcpOk := d.GetOk(fmt.Sprintf("lans.%d.dhcp", lanIndex)); lanDhcpOk {
					lanDhcp := lanDhcp.(bool)
					lan.Dhcp = &lanDhcp
				}
```
which will fail when `Dhcp` is false.